infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/LogEntrySender.java

```diff
@@ -46,6 +46,11 @@ public class LogEntrySender {
 
     private volatile boolean taskActive = false;
 
+    // TODO V2: These values need to be tuned as to not add any unnecessary increase in latency
+    private long waitRetryRead = 100;
+    private final long WAIT_RETRY_READ_INCREMENT = 50;
+    private final long WAIT_RETRY_READ_MAX = 200;
```

Comment:
nit: can you please suffix the constants with "MS" ? That we in a glance we know the Unit of wait time.